# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-viewport/zoom/canvas.html
# Commit: 2de2533b5e62
# Full Hash: 2de2533b5e622f7f6f75e6ad9ec773b1cf997cc8
# Author: Emilio Cobos Álvarez <emilio@crisal.io>
# Date: 2024-09-11 21:40:06
# Regressor Bug: 1909625
# File Overlap Count: 2
# Description:
#   Bug 1909625 - Ignore CSS zoom and text zoom for canvas. r=gfx-reviewers,lsalzman
#   
#   The spec doesn't mention anything about applying them, and other
#   browsers don't, so let's just be consistent...
#   
# ==============================================================================

diff -r c5bedc884f33 -r 2de2533b5e62 testing/web-platform/tests/css/css-viewport/zoom/canvas.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-viewport/zoom/canvas.html	Wed Sep 11 13:46:45 2024 +0000
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<link rel="help" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1909625">
+<link rel="help" href="https://html.spec.whatwg.org/#dom-context-2d-font">
+<link rel="author" href="mailto:emilio@crisal.io" title="Emilio Cobos Álvarez">
+<link rel="author" href="https://mozilla.org" title="Mozilla">
+<link rel="match" href="canvas-ref.html">
+<title>zoom is ignored for canvas</title>
+<style>
+canvas {
+  zoom: 2;
+}
+</style>
+<script>
+document.addEventListener("DOMContentLoaded", () => {
+  const ctx = document.getElementById("canvas").getContext("2d");
+  ctx.font = "48px serif";
+  ctx.fillText(ctx.font, 10, 50);
+});
+</script>
+<canvas id="canvas" width="300" height="100"></canvas>
